src/audio/module_adapter/CMakeLists.txt

```diff
@@ -25,17 +25,20 @@ if(zephyr) ###  Zephyr ###
 		zephyr_library_import(xa_mp3_enc ${CONFIG_CADENCE_CODEC_MP3_ENC_LIB})
 	endif()
 
-	zephyr_include_directories_ifdef(CONFIG_INTEL_MODULES
+	zephyr_include_directories_ifdef(CONFIG_LIBRARY_MANAGER
```

Comment:
Would this not be something like the original CONFIG ? Dont we want to preserve the original Kconfig to select IADK API ?

Reply:
@lgirdwood not sure I understand correctly. Currently my understanding is, that `CONFIG_LIBRARY_MANAGER` is needed always when using the library manager, i.e. for both LLEXT and IADK. While `CONFIG_INTEL_MODULES` is only needed for IADK. So, here I change it to the former to make LLEXT usable without the latter enabled.

Reply:
So we need sof/audio/module_adapter/**iadk**/ directory just with CONFIG_LIBRARY_MANAGER=y?
looks like it should be included only with CONFIG_INTEL_MODULES=y